conformance/utils/config/timeout.go

```diff
@@ -50,8 +50,8 @@ type TimeoutConfig struct {
 func DefaultTimeoutConfig() TimeoutConfig {
 	return TimeoutConfig{
 		CreateTimeout:         60 * time.Second,
-		DeleteTimeout:         10 * time.Second,
-		GetTimeout:            10 * time.Second,
+		DeleteTimeout:         20 * time.Second,
+		GetTimeout:            20 * time.Second,
```

Comment:
haha I actually had a PR ready to push to do this because we also hack it up: https://github.com/ovn-org/ovn-kubernetes/pull/3659/commits/7b071de306fc742f9bf01efc890381c52dd175bb#diff-2894f83ecaa29479d162ded3f17561f1bbe1b4df762c7d65d06ff829c6f5aa9bR47 along with some other cleanup I wanted to do...
but lgtm to this commit since its useful and makes sense.

Reply:
actually nvm because my cleanup commit touches a log of `framework.ExpectNoError` parts, so i will wait for this to go in before i open my PR.. let this merge...